portfolio/src/main/webapp/style.css

```diff
@@ -18,4 +18,9 @@
 
 .bottom-paragraph {
     margin-bottom: 20px;
+}
+
+.top-bottom-margin {
```

Comment:
Naming things is hard! Coming back to this style class in the future, it'll be hard to know what it is used for and if it's safe to adjust the margin (e.g. 10px to 20px).
A name that is indicating the use case (rather than the style) would help.